TemplateBicep/main.bicep

```diff
@@ -103,3 +103,20 @@ module discoveryService './discoveryService.bicep' = if (useDiscoveryService) {
   }
 }
 
+module createDevices 'devices.bicep' = {
```

Comment:
I think this shoudl be conditional

Reply:
I thought so too :)
This is what I found out:
- The function is called nevertheless (no condition on the function launcher)
![image](https://user-images.githubusercontent.com/227765/197528480-96d43479-5ac9-41d6-b5cb-745cef533d8b.png)

- The launcher also does not have a condition:
![image](https://user-images.githubusercontent.com/227765/197528631-e933f6bf-588f-4d83-a060-d28fb88053ec.png)

- The condition is made available as environment variable `DEPLOY_DEVICE` in the function. It only controls if leaf devices are created:
![image](https://user-images.githubusercontent.com/227765/197529084-625f1c0a-7a0f-4e41-872c-be8813295de8.png)

Reply:
As discussed, we will keep the current behavior